src/distributors/distributor-factory.ts

```diff
@@ -1,13 +1,14 @@
-import { AdoptOpenJDKDistributor } from './adoptopenjdk-installer';
+import { AdoptiumDistributor } from './adoptium/adoptium-installer';
 import { JavaBase } from './base-installer';
 import { JavaInstallerOptions } from './base-models';
 import { LocalDistributor } from './local-installer';
```

Comment:
I think we can move local installer to `local` folder as a distributor, but it is up to you

Reply:
Agree, let's move localInstaller to `local` folder. Missed this point

Reply:
Done. Thank you.